### PR TITLE
feat: Created card components

### PR DESCRIPTION
**Pull request by KamiyaGaikwad**

### CODE REVIEW

**Comment by vinc3nati** [comment]
Code is fine, but there are `1,705` file changes for one component, have you used custom fonts i.e. `.ttf` ?

**Comment by KamiyaGaikwad** [comment]
@vinc3nati I have added a font-awesome folder in the commit too because I wanted to use a specific version, that's why it has so many file changes. I haven't used custom font.

**Review by vinc3nati — APPROVED** [review]
Looks good to proceed!

File: components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+
+*{
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+    letter-spacing: 0.5px;
+    font-family: var(--font-family);
+}
+
+#card{
```
Comment (Parul1999) [inline components/Card/card.css]: Styling could be written in class card itself

**Review by Parul1999 — COMMENTED** [review]
try giving units in rem instead of px, especially for height and width

**Comment by KamiyaGaikwad** [comment]
Thank you @Parul1999 for your suggestion, i will do that in my future projects

**Review by Parul1999 — APPROVED** [review]
good to merge

**Pull request merged**